### DISCUSSION TITLE
Is it safe to use Uno.UI.Controls.Legacy library?

### DISCUSSION THREAD

**Discussion by EunjeongChoi414** [unoplatform/uno#15300] [Q&A] [answered]
Our team is trying to use `ProgressRing` provided by Uno platform based on this [Uno doc](https://platform.uno/docs/articles/features/progressring.html#:~:text=Uno.UI.Controls.Legacy)
but because of the naming 'Legacy', I'm not sure if it's safe to use the library? 

Is this library going to be deprecated any time soon? or is it just my bad guess...

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This some outdated documentation that is not clear enough as it mentions the UWP version of the control, and you can use the [`ProgressRing`](https://platform.uno/docs/articles/features/progressring.html#using-the-microsoftuixamlcontrolsprogressring) control directly in WinUI apps. 

It should be working exactly like this: https://playground.platform.uno/#bcce20ad
